### PR TITLE
16.0 mig hr timesheet overtime

### PR DESCRIPTION
**Pull request by robinkeunen**
## Odoo task (if applicable)

https://gestion.coopiteasy.be/web#id=11258&action=475&active_id=492&model=project.task&view_type=form&menu_id=

## Checklist before approval

- [ ] Tests are present (or not needed).
- [ ] Credits/copyright have been changed correctly.
- [ ] Change log snippet is present.
- [ ] (If a new module) Moving this to OCA has been considered.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,6 +22,8 @@ Available addons
 addon | version | maintainers | summary
 --- | --- | --- | ---
 [hr_timesheet_auto_creation](hr_timesheet_auto_creation/) | 16.0.1.0.1 |  | Create weekly timesheets for employees automatically
+[hr_timesheet_project_prevent_creation](hr_timesheet_project_prevent_creation/) | 16.0.1.0.0 | [![carmenbianca](https://github.com/carmenbianca.png?size=30px)](https://github.com/carmenbianca) | Prevent creation of projects and tasks from timesheets.
+[hr_timesheet_sheet_current](hr_timesheet_sheet_current/) | 16.0.1.0.0 |  | Allow to access the current timesheet sheet directly from the menu
```
Comment (remytms) [inline README.md:26]: `hr_timesheet_overtime` seams missing in this list, but is present in the PR.

**Review by remytms — COMMENTED** [review]
Question about the README.md file, the rest seams correct.

File: README.md (hunk 1 above)
Comment (robinkeunen) [reply README.md:26]: I'm pretty sure this is generated by the bot on merge. These lines are actually not added by the migration per se.

Comment (remytms) [reply README.md:26]: ok

**Review by remytms — APPROVED** [review]
LGTM

File: hr_timesheet_overtime/views/resource_view.xml [hunk 2]
```diff
@@ -36,7 +36,7 @@
         <field name="name">resource.overtime.tree</field>
         <field name="model">resource.overtime</field>
         <field name="arch" type="xml">
-            <tree string="Overtime Rate">
+            <tree>
```
Comment (carmenbianca) [inline hr_timesheet_overtime/views/resource_view.xml]: This was done in the prettier/black/isort commit. Is this intentional?

File: hr_timesheet_overtime/views/resource_view.xml [hunk 3]
```diff
@@ -63,7 +63,7 @@
         <field name="name">resource.overtime.rate.tree</field>
         <field name="model">resource.overtime.rate</field>
         <field name="arch" type="xml">
-            <tree string="Overtime Rate" editable="top">
+            <tree editable="top">
```
Comment (carmenbianca) [inline hr_timesheet_overtime/views/resource_view.xml]: idem

**Review by carmenbianca — APPROVED** [review]
lgtm, one comment

**Comment by carmenbianca** [comment]
@robinkeunen can you follow up? :)

File: hr_timesheet_overtime/views/resource_view.xml (hunk 2 above)
Comment (robinkeunen) [reply hr_timesheet_overtime/views/resource_view.xml]: Yes, tree names are not displayed and deprecated : https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst#views

**Comment by remytms** [comment]
\ocabot merge nobump

**Pull request merged**